boards/arm/nucleo_f302r8/doc/nucleof302r8.rst

```diff
@@ -0,0 +1,174 @@
+.. _nucleo_f302r8_board:
+
+ST Nucleo F302R8
+################
+
+Overview
+********
+
+The STM32 Nucleo board provides an affordable and flexible way for users to try
+out new concepts and build prototypes with the STM32 microcontroller, choosing
+from the various combinations of performance, power consumption and features.
+For the compatible boards, the SMPS significantly reduces power consumption in
+Run mode.
+
+The Arduino™ Uno V3 connectivity support and the ST morpho headers allow the
```

Comment:
We recognize trademarks of (other) companies by using an asterisk (``*``) rather than a TM or R mark.  I'm OK if you don't fix them now, but in a quiet period, I'm going to do a pass over all the docs to make this consistent.

Reply:
fixed in commit 22cf2e0d35e